cloud_testing/kotlin_poc/src/main/kotlin/ftl/config/AndroidConfig.kt

```diff
@@ -1,18 +1,25 @@
 package ftl.config
 
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.core.type.TypeReference
 import com.linkedin.dex.parser.DexParser
 import ftl.android.*
 import ftl.gc.GcStorage
 import kotlinx.coroutines.experimental.runBlocking
 
 class AndroidConfig(
+        @field:JsonProperty("app")
         val appApk: String = "",
+        @field:JsonProperty("test")
         val testApk: String = "",
+        @field:JsonProperty("auto-google-login")
         val autoGoogleLogin: Boolean = true,
+        @field:JsonProperty("use-orchestrator")
         val useOrchestrator: Boolean = true,
+        @field:JsonProperty("environment-variables")
         val environmentVariables: Map<String, String> = mapOf(),
+        @field:JsonProperty("directories-to-pull")
         val directoriesToPull: List<String> = listOf(),
-
         rootGcsBucket: String,
         disablePerformanceMetrics: Boolean = true,
```

Comment:
I think the rest of these fields are missing the JsonProperty annotation?

Reply:
Looks like the annotations in the base class cover these ones

Reply:
Great! maybe add a comment since that isn't obvious.

Reply:
Good idea, I'll add one in